pkgs/tools/misc/dsq/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildGo118Module
+, runCommand
+, nix-update-script
+, dsq
+, diffutils
+}:
+
+buildGo118Module rec {
```

Comment:
Ideally this should be done in top-level.

Reply:
Moved, thanks.